Tests/LegacyUnit/OldModel/RegistrationTest.php

```diff
@@ -587,28 +592,26 @@ public function commitToDbCanCreateLodgingsRelation()
             ['lodgings' => [$lodgingsUid]]
         );
         $registration->enableTestMode();
-        $this->testingFramework->markTableAsDirty('tx_seminars_attendances');
-        $this->testingFramework->markTableAsDirty(
```

Comment:
These calls need to stay so the testing framework will clean up the records after running the test.